mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -29,7 +29,8 @@ class PythonLiteralOption(click.Option):
     def type_cast_value(self, ctx, value):
         try:
             return ast.literal_eval(value)
-        except:
+        except Exception as e:
```

Comment:
This is a "catch all" `except`, which you should avoid, because it can hide unexpected errors. In theory, you use an `except` to catch errors that you know might happen, and handle them in a specific way.
When you catch any exception like this (all exceptions in python are subclasses of `Exception`, so you get everything here), something you hadn't expected might happen, and you won't know about it.
The better way to do this is to limit your handling to `except TheExceptionYouKnowMayHappen as e:` and if something else happens, you will then get a clear message. https://stackoverflow.com/questions/14797375/should-i-always-specify-an-exception-type-in-except-statements for some more explanations

Reply:
You're right and got me... Need to find out which error will happen for this case. :D